posthog/queries/util.py

```diff
@@ -8,15 +8,18 @@
 from rest_framework.exceptions import ValidationError
 
 from posthog.client import sync_execute
-from posthog.models.event import DEFAULT_EARLIEST_TIME_DELTA
+from posthog.models.event.event import DEFAULT_EARLIEST_TIME_DELTA
```

Comment:
was this change necessary, given the init file imports everything from `/event/event.py` ?